src/slack/slack.js

```diff
@@ -117,14 +117,9 @@ export class Slack {
           order = this.orders[userId]
           logger.info(`handling user message - user: '${userId}', event text: '${message}', order: ${JSON.stringify(order)}`)
 
-          await this.handleUserMessage(message, userId)
-
-          // check if initial entered link valid. If not, ask for it again.
-          if (this.orders[userId] === null) {
-            say(':exclamation: The links you sent me are invalid.\nIf you want to add a comment, please start by sending the links first and you will be asked for a note later in the process.')
-          }
+          await this.handleUserMessage(message, userId, say)
         } catch (err) {
-          say(':exclamation: Something went wrong, please try again.') // show error to user
+          await say(':exclamation: Something went wrong, please try again.') // show error to user
```

Comment:
I said I'd like to see `Promise.all` for `say+logError`

Reply:
done, please check if it is ok.